.github/workflows/manager_workflow.yml

```diff
@@ -1,6 +1,6 @@
 name: Run manager on runner
 on:
-  push:
+  workflow_dispatch:
```

Comment:
I didn't get this command

Reply:
this just changes how the workflow is triggered in github actions